recipes-ni/crio-support-scripts/crio-support-scripts.bb

```diff
@@ -0,0 +1,33 @@
+SUMMARY = "CompactRIO support files"
+DESCRIPTION = "CompactRIO miscellaneous support files"
+SECTION = "base"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+DEPENDS = "shadow-native pseudo-native niacctbase update-rc.d-native"
+
+PV = "2.0"
+
+
+SRC_URI = "\
```

Comment:
Since we only support x64 in the hardknott mainline, it was an unnecessary complication. I'm going to retain this change, unless you have an objection.